database/migrations/2025_08_22_000003_create_anthropometric_percentiles_table.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('pmieducar.anthropometric_percentiles', function (Blueprint $table) {
```

Comment:
Pode ficar no schema `public` com o tempo queremos remover os demais schemas.

```suggestion
        Schema::create('anthropometric_percentiles', function (Blueprint $table) {
```

Reply:
Obrigado por informar, corrigido!